modules.d/35network-legacy/module-setup.sh

```diff
@@ -4,8 +4,9 @@
 check() {
     local _program
 
-    require_binaries ip dhclient sed awk grep || return 1
-    require_any_binary arping arping2 || return 1
+    require_binaries ip sed awk grep hostname || return 1
+    require_any_binary arping arping2 wicked || return 1
+    require_any_binary dhclient wicked || return 1
```

Comment:
probably install wicked in a separate module, so people can switch it on by purpose.
and maybe create one for dhclient also, so people can omit dhclient.

Reply:
A separate wicked module is currently being developed, disjointed from the legacy module. 
Also, wicked also replaces arping2, not only dhclient in SUSE distros. How about options that can be set in `dracut.conf.d` for a more usercentric choice?